File: src/TemplateModels/JavaLineComment.cs
namespace AutoRest.Java
{
    public class JavaLineComment
    {
        private readonly JavaFileContents contents;

        public JavaLineComment(JavaFileContents contents)
        {
            this.contents = contents;
        }

        public void Line(string text)
        {
            contents.Line(text);
        }
    }
}

File: javagen/src/main/java/com/azure/autorest/model/javamodel/JavaLineComment.java
package com.azure.autorest.model.javamodel;

import com.azure.autorest.util.CodeNamer;

public class JavaLineComment {
    private JavaFileContents contents;

    public JavaLineComment(JavaFileContents contents) {
        this.contents = contents;
    }

    public final void line(String text) {
        contents.line(CodeNamer.escapeComment(text));
    }
}
